mozci/push.py

```diff
@@ -375,6 +375,82 @@ def scheduled_duration(self):
 
         return int(duration / 3600)
 
+    def _check_classification(self, failure_push, classifications):
```

Comment:
Calling this `failure_push` confused me at first because it implies it is the push that caused the failure, when in reality, that is the information we're trying to discover. I'd prefer we simply call it `other` here as well.

Reply:
Sorry about that, I couldn't think of a better name! Renamed to `other` in 1df22fa. I don't like the name so much, but at least it's consistent with get_candidate_regressions.